### DISCUSSION TITLE
Incorporate a registry of users and user groups

### DISCUSSION THREAD

**Discussion by ByZilre** [DirectoryLister/DirectoryLister#1095] [Ideas & Features]
Hello, I am writing to you from Mexico.

and I want to know if an idea can be incorporated into the script: ( https://www.directorylister.com/ )

-My idea is the following: that there is a user registry by the website administrator, where he can create users with permissions only to download files, only downloads.

And the administrator can also create a user that has full permissions.

Then there will only be two groups of users: Admin and User

Admin: all permissions of the Script
User: Only able to download files

Do you think it is possible to carry out this idea?

**Comment by PHLAK** [comment]
Thanks for the suggestion but this is _not_ something we will be implementing in Directory Lister. If you need password protection you can enable authentication on you web server (e.g. Apache, NGINX, etc.).
